Jenkinsfile

```diff
@@ -264,19 +242,14 @@ def getReportLink() {
  return 'Click <a href="$BLUE_OCEAN_URL">here</a> to see the build report'
 }
 
-/*
- *  Change URL Git remote of civihr main repositry to the URL where configured by Jenkins project
- */
-def changeCivihrGitRemote() {
-  def pulledCvhrRepo = sh(returnStdout: true, script: "cd $WORKSPACE; git remote -v | grep fetch | awk '{print \$2}'").trim()
-
-  echo 'Changing Civihr git URL..'
-
-  sh """
-    cd $CIVICRM_EXT_ROOT/civihr
-    git remote set-url origin ${pulledCvhrRepo}
-    git fetch --all
-  """
+def cloneCiviHRRepositories(String envBranch) {
+  for (repo in listCivihrGitRepoPath()) {
+    sh """
+      git clone ${repo.url} ${repo.folder}
+      cd ${repo.folder}
+      git checkout $envBranch || true
```

Comment:
Is the `|| true` here just because the branch might not exist?

Reply:
yes